spec/react_on_rails/generators/install_generator_spec.rb

```diff
@@ -143,7 +143,7 @@
   end
 
   context "without existing application.js or application.js.coffee file" do
-    before(:all) { run_generator_test_with_args([], application_js: false) }
+    before(:all) { run_generator_test_with_args([], application_js: true) }
```

Comment:
we still want to test for it creating an application_js file if one is missing, so we don't want to set this to true for this test, right?